pkg/terror/error_list.go

```diff
@@ -152,6 +152,22 @@ const (
 
 	// pkg/shardddl/optimism
 	codeShardDDLOptimismTrySyncFail
+
+	// pkg/conn
+	codeConnInvalidTLSConfig
+	codeConnRegistryTLSConfig
+
+	// pkg/upgrade
+	codeUpgradeVersionEtcdFail
+
+	// pkg/v1workermeta
+	codeInvalidV1WorkerMetaPath
+
+	// pkg/v1dbschema
+	codeFailUpdateV1DBSchema
```

Comment:
How about coping `Err**` for these codes too?

Reply:
seems not used in v1.x.x. Does it break error code compatibility?

Reply:
if codes exist, it doesn't break compatibility. but if we copied them, may easier for cherry-picking other PRs later?

Reply:
done in https://github.com/pingcap/dm/pull/888/commits/105acfb0144a55172c7d8fa913401a7c412135a4